package.json

```diff
@@ -8,7 +8,7 @@
     "@mui/material": "5.17.1",
     "@mui/utils": "5.17.1",
     "@mui/x-date-pickers": "^7.29.4",
-    "@mui/system": "5.17.1",
+    "@mui/system": "7.1.1",
```

Comment:
I was surprised to see that a 2 major version update didn't seem to break anything in the UI. I don't actually see where we are using this dependency. Can we remove it?

Reply:
https://github.com/DataBiosphere/duos-ui/pull/2895